Bootstrap/app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsFragment.kt

```diff
@@ -0,0 +1,95 @@
+package ar.com.wolox.android.example.ui.home
+
+import android.support.design.widget.FloatingActionButton
+import android.support.v4.widget.SwipeRefreshLayout
+import android.support.v7.widget.DividerItemDecoration
+import android.support.v7.widget.LinearLayoutManager
+import android.support.v7.widget.RecyclerView
+import android.widget.Toast
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.model.News
+import ar.com.wolox.android.example.ui.adapters.NewsAdapter
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import com.facebook.drawee.backends.pipeline.Fresco
+import javax.inject.Inject
+
+
+class NewsFragment @Inject constructor() :  WolmoFragment<NewsPresenter>(), INewsView, SwipeRefreshLayout.OnRefreshListener {
+
+    private lateinit var mRecyclerView: RecyclerView
+    private lateinit var mFragmentRecycleAdapter: NewsAdapter
+    private lateinit var  mLayoutManager: LinearLayoutManager
+    private lateinit var fab:FloatingActionButton
+    private lateinit var mSwipeRefreshLayout: SwipeRefreshLayout
+    private lateinit var  mLineDividerDecoration:DividerItemDecoration
+    private  var N_ITEMS_TO_REFRESH=5
+    private lateinit var newsList: List<News>
+
+    override fun layout(): Int = R.layout.fragment_news
+
+    override fun init() {
```

Comment:
maybe we should divide this in different methods like initRecycler init fab button or something like that.